src/index.js

```diff
@@ -1,27 +1,51 @@
 import './styles.css';
+import Task from './tasks.js';
+import displayTask from './displayTask.js';
+import { editTask, editText, keyPress } from './editTask.js';
 
-const tasks = [
-  {
-    description: 'Practice leetcode algorithms and data structures',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Afterwards set up for codewars algorithms and data structures',
-    completed: false,
-    index: 2,
-  },
-];
+const newTask = new Task();
+const inputTask = document.querySelector('.input-task');
 
-const todoListContainer = document.querySelector('.todo-list');
+function updateUi(id) {
+  const localData = JSON.parse(localStorage.getItem('tasks'));
+  if (localData !== null) {
+    localData.forEach((data) => {
+      /* eslint-disable */
```

Comment:
- [ ] Kindly avoid disabling linters manually, it would be better to ensure your code passes the linter checks, this will help your code more clean and professional  👍🏿

Reply:
Thanks @prolajumokeoni I have made the required changes